<launch>
    <!-- 启动rmtt_sim_node -->
    <node pkg="sunray_rmtt" type="rmtt_sim_node" name="rmtt_sim_node_1" output="screen">
        <param name="uav_id" value="1" />
        <param name="init_pos_x" value="-2.0" />
        <param name="init_pos_y" value="4.0" />
        <param name="init_pos_z" value="0.0" />
    </node>

    <!-- 启动rmtt_sim_node -->
    <node pkg="sunray_rmtt" type="rmtt_sim_node" name="rmtt_sim_node_2" output="screen">
        <param name="uav_id" value="2" />
        <param name="init_pos_x" value="-2.0" />
        <param name="init_pos_y" value="2.0" />
        <param name="init_pos_z" value="0.0" />
    </node>

    <!-- 启动rmtt_sim_node -->
    <node pkg="sunray_rmtt" type="rmtt_sim_node" name="rmtt_sim_node_3" output="screen">
        <param name="uav_id" value="3" />
        <param name="init_pos_x" value="-2.0" />
        <param name="init_pos_y" value="-2.0" />
        <param name="init_pos_z" value="0.0" />
    </node>

    <!-- 启动rmtt_sim_node -->
    <node pkg="sunray_rmtt" type="rmtt_sim_node" name="rmtt_sim_node_4" output="screen">
        <param name="uav_id" value="4" />
        <param name="init_pos_x" value="-2.0" />
        <param name="init_pos_y" value="-4.0" />
        <param name="init_pos_z" value="0.0" />
    </node>

    <!-- 启动rmtt_sim_node -->
    <node pkg="sunray_rmtt" type="rmtt_sim_node" name="rmtt_sim_node_5" output="screen">
        <param name="uav_id" value="5" />
        <param name="init_pos_x" value="2.0" />
        <param name="init_pos_y" value="4.0" />
        <param name="init_pos_z" value="0.0" />
    </node>
</launch>

